src/components/create.vue

```diff
@@ -317,7 +316,9 @@ export default {
             }
           }
         });
-        this.$router.replace("/dashboard");
+        setTimeout(() => {
+            this.$router.replace("/dashboard");
+        }, 50);
```

Comment:
why?

Reply:
When creating an account I often got an error that data in the stores is not available yet, likely due to async dispatching the line before. This is bad style, and it should be promisified

Reply:
saveWallet is a promise. You can simply await the dispatch i think

Reply:
Well, that was too easy xD